pyuvdata/uvcal.py

```diff
@@ -661,6 +661,103 @@ def convert_to_gain(self, delay_convention='minus', run_check=True, check_extra=
         else:
             raise ValueError('cal_type is unknown, cannot convert to gain')
 
+    def get_gains(self, *args):
```

Comment:
The args here should be documented - what should they be?
We also are trying to stay away from args and kwargs when possible. Bryna did a really nice job of revamping the `get_X` methods on UVData - we should try to match that here.